### PR TITLE
KFSPTS-6607 using new attachment icon

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/module/purap/notes-sciquest.tag [hunk 1]
```diff
@@ -192,7 +192,11 @@
                                             </c:if>
                                             <span>
                                                 <c:if test="${kfunc:canViewNoteAttachment(KualiForm.document, attachmentTypeCode)}">
-                                                    <html:image property="methodToCall.downloadBOAttachment.attachment[${status.index}]" src="${ConfigProperties.kr.externalizable.images.url}clip.gif" title="download attachment" alt="download attachment" style="padding:5px;margin-bottom:-10px;" onclick="excludeSubmitRestriction=true"/>
+                                                	<c:set var="mimeTypeCode" value="${note.attachment.attachmentMimeTypeCode}"/>
```
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/module/purap/notes-sciquest.tag]: I think this c:set for the mimeTypeCode should be in the if block above where attachmentTypeCode is being set.

File: src/main/webapp/WEB-INF/tags/module/purap/ponotesSciquest.tag [hunk 2]
```diff
@@ -180,7 +180,11 @@
 									        </c:if>
                                             <span>
                                                 <c:if test="${kfunc:canViewNoteAttachment(KualiForm.document, attachmentTypeCode)}" >
-                                                    <html:image property="methodToCall.downloadBOAttachment.attachment[${status.index}]" src="${ConfigProperties.kr.externalizable.images.url}clip.gif" title="download attachment" alt="download attachment" style="padding:5px" onclick="excludeSubmitRestriction=true"/>
+                                                	<c:set var="mimeTypeCode" value="${note.attachment.attachmentMimeTypeCode}"/>
```
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/module/purap/ponotesSciquest.tag]: I think this c:set for the mimeTypeCode should also be in the if block above where attachmentTypeCode is being set.

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
this is ready for review again

**Pull request merged**